src/testing/runner.ts

```diff
@@ -28,62 +27,32 @@ const fail = (err: string) => {
     process.exit(1);
 };
 
-const run = (...args: any[]) => {
-    const p = execa(...args);
+const run = (processName: string, args?: any[], execaOptions?: any) => {
```

Comment:
Does `execa` include type definitions for it's options?

Reply:
Yes it does.  I was going to do this yesterday but it was causing issues where we used it.  Realised I was being silly so fixed now